.github/workflows/ci.yml

```diff
@@ -0,0 +1,208 @@
+name: CI
+
+# What about CI? For the commits?
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*'
+      - '**'
+    tags-ignore:
+      - '**'
+    paths-ignore:
+      - "**.md"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  validate:
+    name: Code validation
+    runs-on: ubuntu-latest
```

Comment:
This action will fail as it will try to execute everything on the main folder. 

Add this to each job to allow it to run inside the correct folder:

```
 defaults:
      run:
        working-directory: backend    
```

Reply:
But it's working actually. Since we're telling where the `pom.xml` file is.